
# HG changeset patch
# User Maik Broemme <mbroemme@plusserver.de>
# Date 1192552970 25200
# Node ID 4bf08ed71375ff2b6d0c287680af52783f3f1d56
# Parent 456cc6637cbe404fdfe63e3ef16523bfdb1438b0
Add support for Wacom WACF007 and WACF008 to serial pnp driver

Notebook manufacturer seems to built a newer Wacom pen enabled tablet to
recent tablet pcs which are not recognized by the serial pnp driver.

Attached is a patch which makes the newer Wacom WACF007 and WACF008 tablets
useable with the serial driver.  The device is fully compatible with it.

Signed-off-by: Maik Broemme <mbroemme@plusserver.de>
Cc: Andrey Panin <pazke@orbita1.ru>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

committer: Linus Torvalds <torvalds@woody.linux-foundation.org>

--- a/drivers/serial/8250_pnp.c	Tue Oct 16 09:42:50 2007 -0700
+++ b/drivers/serial/8250_pnp.c	Tue Oct 16 09:42:50 2007 -0700
@@ -327,6 +327,8 @@ static const struct pnp_device_id pnp_de
 	{	"WACF004",		0	},
 	{	"WACF005",		0	},
 	{       "WACF006",              0       },
+	{       "WACF007",              0       },
+	{       "WACF008",              0       },
 	/* Compaq touchscreen */
 	{       "FPI2002",              0 },
 	/* Fujitsu Stylistic touchscreens */

